pages/updates/v1-v2.vue

```diff
@@ -0,0 +1,789 @@
+<template>
```

Comment:
these update pages look like they'd better be in nuxt-content

Reply:
implemented in https://github.com/PhotographyLounge/site/pull/1/commits/820e56995adb66a071dee7d1fb49908194f8dd96